CONTRIBUTING.md

```diff
@@ -106,6 +106,27 @@ Some components may require specific setup steps. Please refer to their respecti
 - [Node.js Binding](bindings/nodejs/CONTRIBUTING.md)
 - [Python Binding](bindings/python/CONTRIBUTING.md)
 
+### Run Behaviour Test in MacOS
```

Comment:
How about moving to https://github.com/apache/opendal/tree/main/core/tests/behavior/README.md instead?

Reply:
we can do it like iceberg-rust mention orbstack and podman?

Reply:
> How about moving to https://github.com/apache/opendal/tree/main/core/tests/behavior/README.md instead?

make sense.

Reply:
> we can do it like iceberg-rust mention orbstack and podman?

Good advice! Thanks